### PR TITLE
Kai Roadmap as of July 2024

### PR DESCRIPTION
**Pull request by jwmatthews**

### CODE REVIEW

**Comment by jwmatthews** [comment]
I see a new FOSSA warning, 5 deps are flagged.  This is NOT related to this PR, as in this PR is NOT introducing any changes to deps.   This is an existing issue with the deps used by the website.  

My initial thought is the warnings are not applicable as this repo is not a component to ship to end users, this is just a website to host static content.

File: src/content/blog/kai-roadmap-2024/index.mdx [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+## Purpose
+**What is the purpose of Konveyor AI?**
+Kai intends to improve the economics of re-platforming and refactoring applications to Kubernetes and cloud-native technologies via use of Generative AI leveraging data in Konveyor.
+
+## Maturity
+**Maturity - Early Development - Pre Alpha**
+_Kai is in early stages of development and is NOT suitable for production usage at this time._
+
+- Please see [docs/Evaluation_Builds.md](docs/Evaluation_Builds.md) to learn more about early access preview builds.
```
Comment (savitharaghunathan) [inline src/content/blog/kai-roadmap-2024/index.mdx]: @jwmatthews this is with respect to Kai repo? We might need a fully qualified path here.

File: src/content/blog/kai-roadmap-2024/index.mdx [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+- Contributions are encouraged and most welcome, for more information see [CONTRIBUTING.md](CONTRIBUTING.md)
```
Comment (savitharaghunathan) [inline src/content/blog/kai-roadmap-2024/index.mdx]: Same as above

**Review by savitharaghunathan — CHANGES_REQUESTED** [review]
Minor suggestions to update links. Otherwise looks good to me

File: src/content/blog/kai-roadmap-2024/index.mdx (hunk 1 above)
Comment (savitharaghunathan) [reply src/content/blog/kai-roadmap-2024/index.mdx]: I mean the hyperlink

Comment (jwmatthews) [reply src/content/blog/kai-roadmap-2024/index.mdx]: Thank you!

File: src/content/blog/kai-roadmap-2024/index.mdx (hunk 2 above)
Comment (jwmatthews) [reply src/content/blog/kai-roadmap-2024/index.mdx]: Thank you!

**Comment by jwmatthews** [comment]
Updated links

**Review by rromannissen — APPROVED** [review]
LGTM!

**Review by savitharaghunathan — APPROVED** [review]

**Pull request merged**